### PR TITLE
INVGOV-13430 POST /galleries missing

### PR DESCRIPTION
**Pull request by klauselhai**
https://dev.invotra.com/?url=https://raw.githubusercontent.com/invotra/api/INVGOV-13430.SWAGGER-5.18/api.swagger.yaml

https://io1dev.atlassian.net/browse/INVGOV-13430

### CODE REVIEW

**Review by sam-ludlow — CHANGES_REQUESTED** [review]
Hello, gallery items are actually multi_media CTs.
On the GET we should include the whole multi_media object, on the PUT/POST it should be an object with just UUID (like with taxonomies).

GET:
gallery_items: [
  {
     title: 'sdfvdsf',
     multi_media: { uuid, title, body ... } << full object
  }
]

POST/PUT:
gallery_items: [
  {
     title: 'sdfvdsf',
     multi_media: { uuid: '345345345345' } << just UUID
  }
]

**Comment by klauselhai** [comment]
> Hello, gallery items are actually multi_media CTs.
> On the GET we should include the whole multi_media object, on the PUT/POST it should be an object with just UUID (like with taxonomies).
> 
> GET:
> gallery_items: [
> {
> title: 'sdfvdsf',
> multi_media: { uuid, title, body ... } << full object
> }
> ]
> 
> POST/PUT:
> gallery_items: [
> {
> title: 'sdfvdsf',
> multi_media: { uuid: '345345345345' } << just UUID
> }
> ]

Hi @sam-ludlow 
in example when sending data to drupal, I don’t see multi_media field in this ticket https://io1dev.atlassian.net/browse/INVGOV-13576
or backend guys are missing that field?

**Comment by sam-ludlow** [comment]
Hello, it doesnt matter what its called on drupal services.

We a just calling it multi_media on the API because thats what it is.

Cheers

File: api.swagger.yaml [hunk 1]
```diff
@@ -14245,6 +14245,10 @@ components:
               description: 'The timestamp of the schedule date (in seconds). If present the content is scheduled for a workflow state change, if not, the workflow state will be changed immediately.'
               type: integer
               example: 1510941596
+            gallery_items:
```
Comment (sam-ludlow) [inline api.swagger.yaml:14248]: Please include a description

File: api.swagger.yaml [hunk 2]
```diff
@@ -14498,6 +14503,14 @@ components:
           description: 'The gallery external reference id.'
           type: string
           example: 'ABC123'
+        gallery_items:
+          type: array
+          items:
+            type: object
+            properties:
+              uuid:
```
Comment (sam-ludlow) [inline api.swagger.yaml:14511]: Please include a description & example (see other UUIDs)

**Review by sam-ludlow — CHANGES_REQUESTED** [review]

**Review by sam-ludlow — APPROVED** [review]
Looks good

**Pull request merged**